paddle/fluid/platform/mkldnn_reuse.h

```diff
@@ -230,9 +230,8 @@ class MKLDNNHandler {
     AppendKey(key, std::to_string(groups));
     AppendKey(key, std::to_string(srcdt));
     AppendKey(key, std::to_string(format));
-    AppendKey(key, std::to_string(relu));
+    // AppendKey(key, fuse_activation);
```

Comment:
Please consider removing this AppendKey if it was commented out

Reply:
I forgot to uncomment it out after debugging, thanks.